packages/graphql-eslint-rules/src/operation-naming-conventions.ts

```diff
@@ -120,8 +120,15 @@ const rule: GraphQLESLintRule = {
           );
         }
 
-        const expectedName = expectedPrefix + pascalFilename;
+        if (!filename.startsWith(`${lastDirectory}-`)) {
+          return reportError(
+            context,
+            node,
+            `Filename should start with the component root directory name "${lastDirectory}"`,
```

Comment:
the word component might be overloaded and queries might not always exist in UI only packages?

```suggestion
            `Filename should start with the package directory name: "${lastDirectory}"`,
```